.github/actions/install-dependencies/action.yml

```diff
@@ -1,12 +1,16 @@
 name: 'Install Dependencies'
 description: 'Installs yarn depdencies using cache when available'
+inputs:
+  yarn-hardened-mode:
```

Comment:
This appears to be unused now (yay) can we just remove it, don't want people flipping it without understanding